.github/workflows/development.yml

```diff
@@ -14,4 +14,5 @@ jobs:
     uses: ./.github/workflows/_reusable_app_release.yml
     with:
       fastlane_action: development
+      is_cloud_build: true
```

Comment:
Actually I don't understand why this is required.
There's a default value provided in `_reusable_app_release.yml`.

Reply:
Also I'm not sure if we should put `true` or `false`, any ideas?

Reply:
Not sure if the `default` value only sets the value for the interface to run actions from GitHub? `true` should work here the fast lane action is `develop` and thus builds for our environments.